### PR TITLE
fix(card): update font sizes

### LINKED ISSUE #1729: Card body/footer text size should be 14px by default and header to 16px
Card body and footer text size should be 14px by default and header should be 16px.

Related to #1716

### PR DESCRIPTION
**Pull request by christiemolloy**
Close #1729

### CODE REVIEW

File: src/patternfly/components/Card/card-header.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="pf-c-card__header pf-c-title pf-m-lg{{#if card-header--modifier}} {{card-header--modifier}}{{/if}}"
+<div class="pf-c-card__header pf-c-title pf-m-md{{#if card-header--modifier}} {{card-header--modifier}}{{/if}}"
```
Comment (mcoker) [inline src/patternfly/components/Card/card-header.hbs]: Has @mceledonia looked at this, too? This just changed recently as a result of this conversation with the Integreatly team - https://github.com/patternfly/patternfly-next/issues/626#issuecomment-466500428

File: src/patternfly/components/Card/card.scss [hunk 2]
```diff
@@ -27,6 +29,11 @@
   }
 }
 
+.pf-c-card__body,
+.pf-c-card__footer {
```
Comment (mcoker) [inline src/patternfly/components/Card/card.scss]: Can you separate this out to separate vars?

Comment (mcoker) [reply src/patternfly/components/Card/card.scss]: Looks like you added `--pf-c-card__footer--FontSize`, just aren't using it.

**Comment by christiemolloy** [comment]
@mcoker updated

**Pull request merged**